_data/cloud.yml

```diff
@@ -40,8 +40,9 @@ websites:
       url: https://www.digitalocean.com/
       img: digitalocean.png
       tfa: Yes
+      sms: Yes
       software: Yes
-      doc: https://www.digitalocean.com/company/blog/introducing-two-factor-authentication/
+      doc: https://blog.digitalocean.com/updates-to-digitalocean-two-factor-authentication/
```

Comment:
Instead of a marketing blog announcement, please use this https://www.digitalocean.com/docs/accounts/security/2fa/
It actually tells people how to enable it.

Reply:
Good catch! @mxxcon